### ISSUE TITLE
 [member-ordering] Enforces getter/setter pairs in classes

### ISSUE THREAD

**Issue by splincode** [typescript-eslint/typescript-eslint#4163]
- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.
- [x] I have [read the FAQ](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/TROUBLESHOOTING.md) and my problem is not listed.

**Repro**

```JSON5
{
  "rules": {
    "@typescript-eslint/member-ordering": [
        'error',
        { default: ["field",  "constructor", "get", "set", "method" ] }
     ]
  }
}
```

```TS
class MyFixture {
    constructor() {
    }

    public get a() {return 1;}
    public set a(_) {}

    public get b() {return 1;}
    public set b(_) {}

    public get c() {return 1;}
    public set c(_) {}
}
```

**Expected Result**

```JSON5
{
  "rules": {
    "@typescript-eslint/member-ordering": [
        'error',
        { default: ["field",  "constructor",  "accessors", "method" ] }
     ]
  }
}
```

relative: https://eslint.org/docs/rules/accessor-pairs

**Actual Result**

Invalid result

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `5.4.0` |
| `@typescript-eslint/parser`        | `5.4.0` |
| `TypeScript`                       | `5.4.0` |
| `ESLint`                           | `7.0.0` |
| `node`                             | `16.9.0` |
